package.json

```diff
@@ -100,6 +102,7 @@
     "fork-ts-checker-webpack-plugin": "^7.2.1",
     "husky": "^7.0.4",
     "image-webpack-loader": "^8.1.0",
+    "ioredis-mock": "^7.1.0",
```

Comment:
is it ok that the types are on a different version number ? `"@types/ioredis-mock": "^5.6.0"` vs `"ioredis-mock": "^7.1.0"`

Reply:
both are the latest versions respectively, and it looks like the typings in Definitely Typed for `ioredis-mock` were introduced by somebody outside of the `ioredis-mock` project, so I think that they're just  unrelated